server/config/auth.config.js

```diff
@@ -0,0 +1,5 @@
+secret = {
+    secret: "Our little secret."
```

Comment:
This should be added to the environment file instead of being a separate config.